app/models/allinson_flex/profile_class_context.rb

```diff
@@ -2,6 +2,7 @@
 
 module AllinsonFlex
   class ProfileClassContext < ApplicationRecord
+    self.table_name = 'allinson_flex_profile_classes_contexts'
```

Comment:
Added due to errors thrown when trying to access this table via the rails console.